### PR TITLE
LPS-154811

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

Fixed the bugs in https://github.com/liferay-core-infra/liferay-portal/pull/940#issuecomment-1154170844
Updated based on https://github.com/liferay-core-infra/liferay-portal/pull/877.
This is for ticket https://issues.liferay.com/browse/LPS-154811.
Thanks for checking

### CODE REVIEW

**Comment by catarineaguiar** [comment]
@jeffwu0724 no regressions found.

**Comment by liferay-core-infra** [comment]
it's just a test @rafaprax

File: modules/sdk/gradle-plugins-workspace/build.gradle [hunk 1]
```diff
@@ -119,6 +118,7 @@ dependencies {
 
 	compileInclude files("lib/com.liferay.workspace.bundle.url.codec-1.0.0.jar")
 
+	compileOnly group: 'com.google.code.gson', name: 'gson', version: '2.9.0'
```
Comment (tinatian) [inline modules/sdk/gradle-plugins-workspace/build.gradle:121]: @jeffwu0724 You changed gradle plugin related modules, please check with Peter Shin if you need to include the fake gradle cache.

File: modules/apps/document-library/document-library-document-conversion/bnd.bnd [hunk 2]
```diff
@@ -15,6 +14,7 @@ Import-Package:\
 Require-Capability:\
 	osgi.extender;\
 		filter := "(&(osgi.extender=osgi.configurator)(version>=1.0)(!(version>=2.0)))"
+-fixupmessages: "^Classes found in the wrong directory: \\\\{META-INF/versions/9/";is:=warning
```
Comment (tinatian) [inline modules/apps/document-library/document-library-document-conversion/bnd.bnd:17]: @jeffwu0724 This seems weird, you only changed the compileInclude to compileOnly, why you need to add this? Removing a third lib requires more build requirements does not make sense.

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comment and resend.

Thanks.
Tina.

**Pull request closed without merging**